app/views/shared/_admin_show_team_members.erb

```diff
@@ -10,29 +10,10 @@
   <p class="py-0 my-0">
     <b><%= team_member.position %></b> <%= user_link(team_member.user, true) %>
 
+    <% # Show the badges if the current team_member has an user, and if the currently signed in user is part of the teamwork or can view users, and we are on the admin site. %>
```

Comment:
Grammar nit - also a little bit unclear about meaning here. 
- Current team_member has a user == Current team_member has an account?
- What does 'is part of the teamwork' mean? Part of the team?

```suggestion
    <% # Show the badges if the current team_member has a user, and if the currently signed in user is part of the teamwork or can view users, and we are on the admin site. %>
```

Reply:
A 'teamwork' is the term the website uses for anything with a team, so at the moment just a proposal or event. So yes, 'part of teamwork' should probably be rephrased as 'part of the team for a teamwork'.

A team_member is a many-to-many object, linking a teamwork to a user and keeping a note of their position on that team. So, checking if the team_member has a user is just a check that that relationship is still intact and you are not going to cause a null error when trying to reference that user later.